API/Helpers/PaginationHeader.cs

```diff
@@ -0,0 +1,22 @@
+namespace Helpers
+{
+    public class PaginationHeader
+    {
+        public PaginationHeader(
+            int currentPage,
+            int itemsPerPage,
+            int totalItems,
+            int totalPages
+        ) {
+            this.CurrentPage = currentPage;
```

Comment:
Similar to the comments above, usually server side doesn't need to care the current page number, unless it's a MVC project.